packages/cli/src/commands/setup/i18n/i18n.js

```diff
@@ -66,6 +66,16 @@ export const handler = async ({ force }) => {
               ])
             },
           },
+          {
+            title: 'Sync yarn.lock and node_modules',
```

Comment:
@thedavidprice  I know you remove the sync Yarn because it was not necessary 
> I had a couple things I wanted to test locally (e.g. if the "sync Yarn" step was necessary as it was for Tailwind; in this case it was not).

But in my local test, I have the error `Module not found: Error: Can't resolve i18n ...` it probably happens when you use --force option 
Also, I think it's nice to have the module already install after the setup command. What do you think?

Reply:
> But in my local test, I have the error Module not found: Error: Can't resolve i18n ...

Hmm, I tested several cases locally and had no errors. In Tailwind, where this originated, after installing we used the Tailwind CLI to set up config, which needed the `yarn --check-files` in the case of --force.

I'll take a look now and see if I can confirm success.

Reply:
I also use `yarn rwt link path/to/project` but it's probably something weird in my local setup. 😋